<<<<<<< Updated upstream
This file is part of MXE. See LICENSE.md for licensing information.

Contains ad hoc patches for cross building.

From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mark Brand <mabrand@mabrand.nl>
Date: Mon, 14 Apr 2025 14:40:55 +0200
Subject: [PATCH 1/1] fix pkg-config


diff -Nru lapack-3.12.1-orig/CMakeLists.txt lapack-3.12.1-patched/CMakeLists.txt
--- lapack-3.12.1-orig/CMakeLists.txt	2024-12-03 06:39:11.000000000 -0500
+++ lapack-3.12.1-patched/CMakeLists.txt	2025-01-20 05:42:26.733216652 -0500
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.13)
 
-project(LAPACK C)
+project(LAPACK C Fortran)
 
 set(LAPACK_MAJOR_VERSION 3)
 set(LAPACK_MINOR_VERSION 12)
